etcdserver/api/rafthttp/metrics.go

```diff
@@ -71,6 +71,68 @@ var (
 		[]string{"From"},
 	)
 
+	snapshotSend = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "etcd",
+		Subsystem: "network",
+		Name:      "snapshot_send",
```

Comment:
This is for successful sends only, right?  Maybe just clarify that in the help string so that it's clear that the total sent count would be this + `snapshot_send_failures`?  Alternatively we could name this "snapshot_send_success", but that seems a bit too verbose..

Reply:
Agree. Will suffix `success`.